package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Hey @djfarrelly! Curious if this needs to live in the project root or could perhaps be moved to the `package/node` directory?